cmd/execution/main.go

```diff
@@ -248,6 +248,7 @@ func main() {
 
 			// Needed for gRPC server, make sure to assign to main scoped vars
 			events = storage.NewEvents(node.DB)
+			serviceEvents := storage.NewServiceEvents(node.DB)
```

Comment:
good choice to keep them separate than events, in case we need to do special handling for archival nodes.